README.md

```diff
@@ -70,7 +70,8 @@ It's intended to be the Wayland equivalent of an xrandr GUI, like [ARandR].
 Sway, like i3, doesn't save any settings unless you put them in the config
 file. See man `sway-output`. If you want to have multiple configurations
 depending on the monitors connected, you'll need to use an external program
-like [kanshi]. Integration with that and other external daemons is planned.
+like [kanshi] or [way-displays]. Integration with that and other external
+daemons is planned.
```

Comment:
I'm happy to say:

"other external daemons such as [way-displays] is planned."